roles/pulp_api/tasks/main.yml

```diff
@@ -62,17 +62,30 @@
   when:
     - (ansible_version.major > 2) or (ansible_version.major == 2 and ansible_version.minor >= 9)
 
+- name: Get database config
```

Comment:
This isn't very descriptive.

More like `Get the database fields key from the pulp_database_config node`

Add a comment that it will include the newline.